### ISSUE TITLE
Please provide an URL list used by the container for use with containerize.

### ISSUE THREAD

**Issue by famewolf** [containers-everywhere/contain-google#128]
I would prefer not to install a bunch of extensions to have google container, facebook container, Amazon Container etc.    Containerize allows for a csv list where you can specify the domain names and what container to use.  Can the list be made available?  google.com is obvious.  gmail.com is less obvious and youtube.com is often missed...etc.

**Comment by flexagoon** [comment]
The full list is here:

https://github.com/containers-everywhere/contain-google/blob/master/background.js

**Comment by joakim** [comment]
Good idea! I extracted 409 URLs from the Facebook Container and Google Container extensions and massaged it into CSV that can be fed into the [Containerise](https://addons.mozilla.org/en-US/firefox/addon/containerise/) extension. I called the containers `Evil Alphabet` and `Evil Meta`, just to get the point across.

https://gist.github.com/joakim/426e56f489fe905dd939efaccf1ff07a

You're welcome. Containerise is _not_ easy to work with, so it took some time.

If you want to say thanks, feel free to [buy me a coffee](https://ko-fi.com/joakims) :)

**Comment by elesto** [comment]
The issue with containerise is it currently doesn't have the option to not override other containers.
